app/models/piece.rb

```diff
@@ -19,4 +19,19 @@ def move_to!(row, col)
     raise "Out of bounds" if row < 0 || row > 7 || col < 0 || col > 7
     update(row: row, col: col)
   end
+
+  def capture_piece(row, col)
+    other_piece = game.piece_at(row, col)
+    return true if game.path_obstructed? && other_piece.user != user
```

Comment:
Makes sense.  I was putting the code together assuming we would make the change when the `path_obstructed?` method got merged.  I see that we want to make it work first and then change it.  Its a little weird, since I worked on the `path_obstructed?` method with Marcia, so I am expecting to incorporate it.  I will go ahead and make the change as you have above though.